doc/src/modules/polys/basics.rst

```diff
@@ -14,6 +14,319 @@ polynomials within SymPy. All code examples assume::
     >>> x, y, z = symbols('x,y,z')
     >>> init_printing(use_unicode=False, wrap_line=False, no_global=True)
 
+Basic concepts
+==============
+
+Polynomials
+-----------
+
+Given a family `(x_i)` of symbols, or other suitable objects,
+expressions derived from them by repeated addition, subtraction
+and multiplication are called *polynomial expressions in the
+generators* `x_i`.
+
+By the distributive law it is possible to perform
+multiplications before additions and subtractions.
+The products of generators thus obtained are called
+*monomials*. They are usually written in the form
+`x_1^{\nu_1}x_2^{\nu_2}\cdots x_n^{\nu_n}` where the exponents `\nu_i`
```

Comment:
Backquotes automatically do inline math. You only need a `.. math::` for display math.